channeldb/migration12/invoices.go

```diff
@@ -345,71 +346,90 @@ func deserializeHtlcs(r io.Reader) (map[CircuitKey]*InvoiceHTLC, error) {
 //
 // nolint: dupl
 func serializeInvoice(w io.Writer, i *Invoice) error {
-	if err := wire.WriteVarBytes(w, 0, i.Memo[:]); err != nil {
-		return err
-	}
-	if err := wire.WriteVarBytes(w, 0, i.Receipt[:]); err != nil {
-		return err
-	}
-	if err := wire.WriteVarBytes(w, 0, i.PaymentRequest[:]); err != nil {
-		return err
-	}
-
-	if err := binary.Write(w, byteOrder, i.FinalCltvDelta); err != nil {
-		return err
-	}
-
-	if err := binary.Write(w, byteOrder, int64(i.Expiry)); err != nil {
-		return err
```

Comment:
Maybe this shouldn't have been copied in the previous commit?

Reply:
done